internal/pkg/formatter/default.go

```diff
@@ -293,9 +293,22 @@ func printMetadata(w io.Writer, result scan.Result, links []string, isRego bool)
 }
 
 // nolint
-func highlightCode(b formatters.ConfigurableFormatter, result scan.Result) error {
+func highlightCode(b formatters.ConfigurableFormatter, result scan.Result, theme string) error {
 
-	code, err := result.GetCode()
+	codeOpts := []scan.CodeOption{
+		scan.OptionCodeWithTruncation(true),
+	}
+
+	switch theme {
+	case "dark":
+		codeOpts = append(codeOpts, scan.OptionCodeWithDarkTheme())
+	case "light":
+		codeOpts = append(codeOpts, scan.OptionCodeWithLightTheme())
+	default:
+		codeOpts = append(codeOpts, scan.OptionCodeWithTheme(theme))
```

Comment:
ignore previous question

Reply:
but what if I write junk in here - does it fallback in defsec?

Reply:
Yeah it falls back to a default dark one.

Reply:
Thought i'd leave it undocumented so we can play with it, but tell people to only use dark/light